src/Relude/File.hs

```diff
@@ -68,6 +77,7 @@ readFileText :: MonadIO m => FilePath -> m Text
 readFileText = liftIO . T.readFile
 {-# SPECIALIZE readFileText :: FilePath -> IO Text #-}
 {-# INLINE     readFileText #-}
+{-# WARNING readFileText ["'readFileText' depends on the system's locale settings and can throw unexpected exceptions.", "Use 'readFileBS' instead."] #-}
```

Comment:
Using list of strings here to display on different lines as written in docs:

* https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/pragmas.html#warning-and-deprecated-pragmas

Example:

![image](https://user-images.githubusercontent.com/4276606/163408869-a28715ee-7ee1-4c6d-bdd0-bc963b654226.png)